src/ser.rs

```diff
@@ -107,9 +107,6 @@ pub enum Error {
     /// attempted where the key of a map was not a string.
     KeyNotString,
 
-    /// Keys in maps are not allowed to have newlines.
-    KeyNewline,
```

Comment:
Ah unfortunately I think *this* is a breaking change (removing an existing variant), but I think it's ok to basically leave a "dead variant" here. Maybe `#[doc(hidden)]` it?